### PR TITLE
Implement remaining weather effects

### PR DESCRIPTION
**Pull request by GRardB**
https://user-images.githubusercontent.com/748180/204252162-4e9d6414-2027-43a2-a820-deb74aed9551.mov

https://user-images.githubusercontent.com/748180/204252192-f48ef65f-5720-47b3-85d1-36625bdf67b3.mov

https://user-images.githubusercontent.com/748180/204252198-824c6285-2b9b-4584-be6f-36c1f76e12cd.mov

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -7,12 +7,14 @@ module.exports = {
...
-        "slide-down": "slide-down 150ms ease-in-out",
-        "slide-up": "slide-up 150ms ease-in-out",
-        "pop-down": "pop-down 150ms ease-out forwards",
-        "pop-up": "pop-up 150ms ease-out forwards",
         "dialog-hide": "dialog-hide 150ms ease-out",
         "dialog-show": "dialog-show 150ms ease-out",
+        "pop-down": "pop-down 150ms ease-out forwards",
+        "pop-up": "pop-up 150ms ease-out forwards",
+        "slide-bg-right-slow": "slide-bg-right-slow 30s linear infinite",
+        "slide-bg-right-fast": "slide-bg-right-fast 15s linear infinite",
```
Comment (stephan281094) [inline tailwind.config.js]: These two can use the same keyframe:
```suggestion
        "slide-bg-right-slow": "slide-bg-right 30s linear infinite",
        "slide-bg-right-fast": "slide-bg-right 15s linear infinite",
```

File: tailwind.config.js [hunk 2]
```diff
@@ -76,6 +81,22 @@ module.exports = {
...
+          },
+        },
+        "slide-bg-right-fast": {
+          from: {
+            backgroundPosition: "0px 0px",
+          },
+          to: {
+            backgroundPosition: "1000px 0px",
+          },
+        },
```
Comment (stephan281094) [inline tailwind.config.js]:
```suggestion
        "slide-bg-right": {
          from: {
            backgroundPosition: "0 0",
          },
          to: {
            backgroundPosition: "1000px 0",
          },
        },
```

**Review by stephan281094 — CHANGES_REQUESTED** [review]
Love the `useBackgroundEffect` refactor!

Could you maybe also remove the `.DS_Store` files?

File: tailwind.config.js (hunk 1 above)
Comment (GRardB) [reply tailwind.config.js]: D'OH

**Review by stephan281094 — APPROVED** [review]

**Pull request merged**